testing/gnome-online-accounts/APKBUILD

```diff
@@ -1,40 +1,37 @@
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gnome-online-accounts
-pkgver=3.26.0
-pkgrel=1
-pkgdesc="client for GNOME web services"
-url="https://www.gnome.org/"
+pkgver=3.32.0
+pkgrel=0
+pkgdesc="Single sign-on framework for GNOME"
+url="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
 # limited by webkit2gtk
```

Comment:
It looks like [webkit2gtk](https://github.com/alpinelinux/aports/blob/master/community/webkit2gtk/APKBUILD#L9) is now supported on all architectures.

```suggestion
```